app/views/editor/thang/ThangTypeEditView.coffee

```diff
@@ -716,6 +717,16 @@ module.exports = class ThangTypeEditView extends RootView
     modal = new ExportThangTypeModal({}, @thangType)
     @openModalView(modal)
 
+  # Run it in the editor/thang/<thang-type> view by inputting the following:
+  # ```
+  # currentView.normalizeColorsForCustomization()
+  # ```
+  # into the console. Used to normalize the shape colors for Ozaria Hero A and B
+  # to support customization.
+  normalizeColorsForCustomization: ->
+    replaceRgbaWithCustomizableHex(@thangType.attributes.raw.shapes)
```

Comment:
In general I like functions without side effects like modifying the argument. In this case, I think it is especially useful because it is hard for a new reader to grok that the `@thangType.attributes.raw.shapes` is being modified here, and then used in `@treema.set('raw', @thangType.get('raw'))` afterwards. 

If the function worked on a copy of the value passed in, and returned the modified copy, it would be much easier to read for me.

Reply:
I think I have made this clearer now. It looks a bit janky but mutation is gone!